### ISSUE TITLE
Cloud Emojis Missing on Search

### ISSUE THREAD

**Issue by kenglub** [GRA0007/emoji-menu#5]
Cloud emoji mixes aren’t appearing when the emoji is entered into the search bar on ios 14. ☁️ < This guy
Some cloud mixes will appear for other searches but not on its own search.

**Comment by GRA0007** [comment]
Yep, there are actually quite a few that haven't been added to the database yet, I'm still working on it. Thanks for the heads up 😄
